jupyter-notebooks/subscriptions_api_tutorial/Biomass_Proxy_PV_Subs_Ohio.ipynb

```diff
@@ -0,0 +1,736 @@
+    "    response = requests.post(BASE_URL, data=json.dumps(subscription_desc), auth=auth, headers=headers)\n",
+    "    print(response)\n",
+    "    subscription_id = response.json()['id']\n",
+    "    print(subscription_id)\n",
+    "    subscription_url = BASE_URL + '/' + subscription_id\n",
+    "    return subscription_url\n",
+    "\n",
+    "pv_subscription = subscribe_pv(subscription_desc, auth, headers)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "14178e53",
+   "metadata": {},
+   "source": [
+    "## Using the data from Planet"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b353d9a6",
+   "metadata": {},
+   "source": [
+    "### Get the subscription\n",
+    "\n",
+    "Here we check the status of the subscription we created. When it has the status `completed`, it'll be delivered to the bucket, above.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "b56bf94f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'completed'"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Get the subscription ID - GCP bucket directory name where the data will be delivered\n",
+    "sub_id = requests.get(pv_subscription, auth=auth).json()['id']\n",
+    "# for example: sub_id = \"69b83d57-c885-464a-9a94-5c1d2bfacbd4\"\n",
+    "\n",
+    "# Use sub_id to get the subscription state\n",
+    "def get_sub_status(sub_id, auth):\n",
+    "    sub_url = BASE_URL + '/' + sub_id\n",
+    "    sub_status = requests.get(sub_url, auth=auth).json()['status']\n",
+    "    return sub_status\n",
+    "\n",
+    "# Note: Running the subscription may take a while depending \n",
+    "# on the size of your AOI and the time range.\n",
+    "sub_status = get_sub_status(sub_id, auth)\n",
+    "sub_status\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "4519bdd0",
+   "metadata": {},
+   "source": [
+    "Use the pandas library to read the CSV data into a DataFrame called `df`. Filter only valid data is retained in the DataFrame for further analysis or processing.\n",
+    "\n",
+    "\n",
+    "Next, you can retrieve data in CSV format from a specified URL using an API request. Then read the CSV data into a Pandas DataFrame. Use `item_datetime` in a datetime format and removes the timezone information. You can filter the DataFrame to include only rows where the month of `item_datetime` falls between May and November. You can also filter the DataFrame to keep only rows with valid data, excluding any rows with null or zero values for `bp.band-1.valid_percent`, as well as rows with a status of `QUEUED`. You have the option to assign a title value to the DataFrame from a subscription description. Finally, view the first few rows of the DataFrame to examine the data.\n",
```

Comment:
Similar to the previous notebook- here when we mention "bp.band-1.valid_percent" I think we should link to the docs so that folks know what we're talking about

Reply:
done. Thanks.